android.pri

```diff
@@ -1,9 +1,33 @@
 include($$PWD/libs/qtandroidserialport/src/qtandroidserialport.pri)
 message("Adding Serial Java Classes")
 QT += androidextras
+
 ANDROID_PACKAGE_SOURCE_DIR = $$PWD/android
+
+exists($$PWD/custom/android) {
+    message("Merging $$PWD/custom/android/ -> $$PWD/android/")
+
+    ANDROID_PACKAGE_SOURCE_DIR = $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR
+    android_source_dir_target.target = android_source_dir
+    PRE_TARGETDEPS += $$android_source_dir_target.target
+    QMAKE_EXTRA_TARGETS += android_source_dir_target
+
+    android_source_dir_target.commands = $$QMAKE_MKDIR $$ANDROID_PACKAGE_SOURCE_DIR && \
+            $$QMAKE_COPY_DIR $$PWD/android/* $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR && \
+            $$QMAKE_COPY_DIR $$PWD/custom/android/* $$OUT_PWD/ANDROID_PACKAGE_SOURCE_DIR && \
+            $$QMAKE_STREAM_EDITOR -i \"s/package="org.mavlink.qgroundcontrol"/package=\"$$QGC_ORG_DOMAIN.qgroundcontrol\"/\" $$ANDROID_PACKAGE_SOURCE_DIR/AndroidManifest.xml
```

Comment:
This hi-jacks `QGC_ORG_DOMAIN` for the purpose of identifying the custom QGC app for android. Do you want me to define `QGC_ANDROID_PACKAGE` instead to be explicit?

Reply:
Yeah I think a `QGC_ANDROID_PACKAGE` would be better.

Reply:
Done: https://github.com/mavlink/qgroundcontrol/pull/8970/commits/9adc28bd0f454502fa27f8af7fc441bbee318a81